tests/robot-tests/tests/admin/bau/release_status.robot

```diff
@@ -115,8 +115,12 @@ Add text block with content to Test section two
     user adds text block to editable accordion section    Test section two    ${RELEASE_CONTENT_EDITABLE_ACCORDION}
     user adds content to autosaving accordion section text block    Test section two    1
     ...    Test section two text    ${RELEASE_CONTENT_EDITABLE_ACCORDION}
+    user scrolls to accordion section    View related dashboard(s)    id:data-accordion
```

Comment:
I'll still approve, but I don't understand why this scrolling is necessary before reloading the page?

If we're reloading the page, I wouldn't expect scrolling to an element first to have any effect.

Reply:
What I observed is sometimes page is getting frozen intermittently in pipelines. There are couple of issues behind the failure - 1 scrolling issue and 2 stale element issue. Anyway, I will try vice versa of both the steps, if it works then I will push the changes.